metrics/src/main/java/io/opencensus/metrics/LabelKey.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2018, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opencensus.metrics;
```

Comment:
Do you think we should put the whole metrics API in one package?

Reply:
For now all the classes are in the same proto file, and all the Java classes generated from it will be under the same package. I think we can start by putting everything under one package to be consistent with that. We can move the APIs to sub-packages later if needed.